build.psm1

```diff
@@ -601,7 +601,7 @@ function Restore-PSModuleToBuild
     Restore-PSModule -Destination $modulesDir -Name @(
         # PowerShellGet depends on PackageManagement module, so PackageManagement module will be installed with the PowerShellGet module.
         'PowerShellGet'
-    )
+    ) -SourceLocation "https://www.powershellgallery.com/api/v2/"
```

Comment:
This call to Restore-PSModule can be combined with the call at line # 607. The name is an array.

```Powershell

Restore-PSModule -Destination $modulesDir -Name @( 
         'Microsoft.PowerShell.Archive','PowerShellGet'
     ) -SourceLocation "https://www.powershellgallery.com/api/v2/"